### ISSUE TITLE
[Clearer] Focus support 

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#1542]
Select clearer should be focusable. 
It's possible to get focus by replacing `<a>` by `<button>`, but for now, pressing space / enter while focusing will open select instead of clearing the field.

**Comment by jeremie-lucca** [comment]
Fixed in new Simple / Multiple Selects 😊
Will not be fixed in luSelect as it's going to be deprecated soon.
